packages/vm/src/runTx.ts

```diff
@@ -30,11 +30,6 @@ const debugGas = createDebugLogger('vm:tx:gas')
  * @ignore
  */
 export async function runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
-  // tx is required
```

Comment:
Is there some reason we're removing this check?

Reply:
Yep, see comment from Jochem here: https://github.com/ethereumjs/ethereumjs-monorepo/pull/2233#discussion_r956591974
basically `tx` is a non-optional option so `opts.tx` should never be undefined here